app/components/moderation-list/style.scss

```diff
@@ -71,6 +72,29 @@
   }
 }
 
+@media screen and (max-width: $screen-medium-small) {
+  .moderation-list-heading {
+    .btn {
+      font-size: 14px;
```

Comment:
It would be awesome to have this sort of responsive `font-size` for the entire app -- it looks weird when just the list heading shrinks and everything above and below it looks huge in comparison.

Would it be feasible to do something like that everywhere without repetitive code? Maybe consistently use `em`/`rem` for `font-size`, and a single media query on `html` to set `font-size` based on the viewport width? Or does that seem like a bad idea (or too much to fit in this ticket)?

Reply:
@laurenbarker do you have thoughts on this?

Reply:
It should probably be another ticket. Definitely worth doing though so I'll go ahead and add one.